integreat_compass/cms/models/interactions/vote.py

```diff
@@ -11,7 +11,7 @@ class Vote(AbstractBaseModel):
     Data model representing a Vote.
     """
 
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    creator = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
I think if a user is deleted, his/her votes should definitely stay, as otherwise if someone is removed from the "gremium" some offers do not have enough votes anymore.